apps/contracts-stylus/src/lib.rs

```diff
@@ -38,99 +37,13 @@ impl GameOfLifeNFT {
     pub fn token_uri(&self, token_id: U256) -> Result<String, Vec<u8>> {
         let seed = token_id.as_limbs()[0];
         let size = 32;
-        let generations = 64; //64 works with decent performances, 128 works but the browser is slow ,  512 & 1024 out of gas error
+        let generations = 64;
         let cell_size = 4;
-
-        let mut svg = String::with_capacity(size * size * 32);
-        svg.push_str(
-            r#"<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><style>
-            @keyframes gol_pulse {
-                0% {opacity:0}
-                20% {opacity:1}
-                80% {opacity:1}
-                100% {opacity:0}
-            }
-            @keyframes gol_last {
-                0% {opacity:0}
-                20% {opacity:1}
-                100% {opacity:1}
-            }
-            .gol-cell {fill:#000;opacity:0;animation:gol_pulse 500ms forwards}
-            .gol-cell.last {animation:gol_last 500ms forwards}
-            </style>"#,
+        let json = format!(
+            "{{\"seed\":{},\"size\":{},\"generations\":{},\"cell_size\":{}}}",
+            seed, size, generations, cell_size
```

Comment:
Well, while isn't wrong the approach there are two aspects:
- The purpose to show the performance differences between Stylus and Solidity is gone
- The rendering moves from on-chain to the client implementation. There is value on having the artwork in an immutable ledger ;)